estimator/util.py

```diff
@@ -230,3 +275,7 @@ def batch_estimate(params, algorithm, jobs=1, log_level=0, **kwds):
         ret[x][f] = res[f, x]
 
     return ret
+
+
+def log2(x):
```

Comment:
Do we need this?

Reply:
It's heavily used in the MITM attack. Got tired of writing log(x, 2), which looks kinda ugly when x is some longer expression. I thought it might be useful more generally, but if you think it's superfluous, I can also remove it and either place it in lwe_brute_force.py or replace it with log(x, 2) everywhere.

Reply:
It's purely aesthetics on my part. I'd perhaps define it inside MITM? But it's no big deal either way.